vlm/src/main/scala/geotrellis/contrib/vlm/avro/GeotrellisRasterSource.scala

```diff
@@ -109,14 +112,14 @@ class GeotrellisRasterSource(
   def resample(resampleGrid: ResampleGrid[Long], method: ResampleMethod, strategy: OverviewStrategy): RasterSource = {
     val resampledGridExtent = resampleGrid(this.gridExtent)
     val closestLayerId = GeotrellisRasterSource.getClosestResolution(sourceLayers.toList, resampledGridExtent.cellSize, strategy)(_.metadata.layout.cellSize).get.id
-    new GeotrellisResampleRasterSource(attributeStore, uri, closestLayerId, sourceLayers, resampledGridExtent, method, targetCellType)
+    new GeotrellisResampleRasterSource(attributeStore, dataPath, closestLayerId, sourceLayers, resampledGridExtent, method, targetCellType)
   }
 
   def convert(targetCellType: TargetCellType): RasterSource =
-    new GeotrellisRasterSource(attributeStore, uri, layerId, sourceLayers, bandCount, Some(targetCellType))
+    new GeotrellisRasterSource(attributeStore, dataPath, layerId, sourceLayers, bandCount, Some(targetCellType))
 
   override def toString: String =
-    s"GeoTrellisRasterSource($uri,$layerId)"
+    s"GeoTrellisRasterSource(${dataPath.toString},$layerId)"
```

Comment:
TIP: you already overrided to string, in the string interpolator `toString` method would be invoked to convert object into string already.